apps/python_bindings/sssp/sssp.py

```diff
@@ -0,0 +1,21 @@
+import graphit
```

Comment:
Could you add an acknowledgement to the UW folks for these files in the comments? Take a look at their fork of GraphIt to figure out who wrote what parts?